sale_order_alternate_name/models/sale_order.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    alternate_name_description = fields.Char(
+        compute="_compute_alternate_name_description"
+    )
+    alternate_name = fields.Char(compute="_compute_alternate_name", store=False)
+
+    @api.depends("partner_id", "name", "bike_location", "alternate_name_description")
+    def _compute_alternate_name(self):
+        bike_location_mapping = dict(
+            self.fields_get("bike_location", "selection")["bike_location"]["selection"]
+        )
+        for rec in self:
+            if rec.alternate_name_description:
```

Comment:
I think it can bring us issues showing the task on the calendar on orders without description products and with service products, we still need the partner info on the task. We can discuss that